Translations/zh_CN.po

```diff
@@ -1035,110 +1031,95 @@ msgstr "通向"
 
 #: Source/cursor.cpp:256
 msgid "The Unholy Altar"
-msgstr "邪恶的祭坛"
+msgstr "不洁祭坛"
 
 #: Source/cursor.cpp:258
 msgid "level 15"
 msgstr "15级"
 
-#: Source/diablo.cpp:119
+#: Source/diablo.cpp:116
 msgid "I need help! Come Here!"
 msgstr "我需要帮助！过来！"
 
-#: Source/diablo.cpp:120
+#: Source/diablo.cpp:117
 msgid "Follow me."
 msgstr "跟着我。"
 
-#: Source/diablo.cpp:121
+#: Source/diablo.cpp:118
 msgid "Here's something for you."
 msgstr "这是给你的。"
 
-#: Source/diablo.cpp:122
+#: Source/diablo.cpp:119
 msgid "Now you DIE!"
 msgstr "现在你死定了！"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:886
+#: Source/diablo.cpp:855
 msgid "Options:\n"
 msgstr "选项: \n"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:887
+#: Source/diablo.cpp:856
 msgid "Print this message and exit"
 msgstr "打印此消息并退出"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:888
+#: Source/diablo.cpp:857
 msgid "Print the version and exit"
 msgstr "打印版本并退出"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:889
+#: Source/diablo.cpp:858
 msgid "Specify the folder of diabdat.mpq"
 msgstr "指定diabdat.mpq的文件夹"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:890
+#: Source/diablo.cpp:859
 msgid "Specify the folder of save files"
 msgstr "指定保存文件的文件夹"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:891
+#: Source/diablo.cpp:860
 msgid "Specify the location of diablo.ini"
 msgstr "指定diablo.ini的位置"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:892
+#: Source/diablo.cpp:861
 msgid "Specify the location of the .ttf font"
 msgstr "指定.ttf字体的位置"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:893
+#: Source/diablo.cpp:862
 msgid "Specify the name of a custom .ttf font"
 msgstr "指定自定义.ttf字体的名称"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:894
+#: Source/diablo.cpp:863
 msgid "Skip startup videos"
 msgstr "跳过启动视频"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:895
+#: Source/diablo.cpp:864
 msgid "Display frames per second"
 msgstr "每秒显示帧数"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:896
+#: Source/diablo.cpp:865
 msgid "Run in windowed mode"
 msgstr "在窗口模式下运行"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:897
+#: Source/diablo.cpp:866
 msgid "Enable verbose logging"
 msgstr "启用详细日志记录"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:898
+#: Source/diablo.cpp:867
 msgid "Force spawn mode even if diabdat.mpq is found"
-msgstr "即使找到diabdat.mpq，也强制生成模式"
-
-#. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:899
-msgid "Record a demo file"
-msgstr ""
+msgstr "即使找到diabdat.mpq，也使用强制生成模式"
 
 #. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:900
-msgid "Play a demo file"
-msgstr ""
-
-#. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:901
-msgid "Disable all frame limiting during demo playback"
-msgstr ""
-
-#. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:902
```

Comment:
Looks like you might have reverted to an older version, could you run "Update From Code" and see if that imports the missing bits?

Reply:
The pot file not been updated. I can't find those text in
https://github.com/diasurgical/devilutionX/blob/master/Translations/devilutionx.pot
😳 I think the pot file should be updated to 2021-08-09 14:40+0800. The provided devilutionx.pot version is 2021-07-31 17:03+0200.

Reply:
Fair enough, usually I get an option to update directly from the code:
![image](https://user-images.githubusercontent.com/204594/129175165-045a0450-7321-49ca-ad49-ab830ce12f33.png)